test/e2e/resize_nodes.go

```diff
@@ -324,53 +350,49 @@ func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replica
 	default:
 		Failf("This test is not supported for provider %s and should be disabled", testContext.Provider)
 	}
-	iptablesRule := fmt.Sprintf("OUTPUT --destination %s --jump REJECT", master)
+	return master
+}
+
+func getNodeExternalIP(node *api.Node) string {
```

Comment:
can you clarify that this returns ip+port for ssh? either via comment or name change